docs/source/migration.mdx

```diff
@@ -44,115 +46,113 @@ Apollo Server 3 no longer contains this built-in integration. However, you can s
 
 We hope to add more fully-integrated subscription support to Apollo Server in a future version.
 
+**Please note that the `subscriptions-transport-ws` library is not actively maintained.** However, as this is written, this protocol is exclusively supported by both GraphQL Playground and Apollo Sandbox. An alternative, actively-maintained [`graphql-ws`](https://www.npmjs.com/package/graphql-ws) package exists which we intend to support in the future.
```

Comment:
How about:

**Please note that the `subscriptions-transport-ws` library is not actively maintained.** An alternative, actively-maintained [`graphql-ws`](https://www.npmjs.com/package/graphql-ws) package exists which we intend to support in the future. These libraries implement different protocols for GraphQL subscriptions over websockets, so you will need to adjust your client to support `graphql-ws`. At the time of release of Apollo Server 3, the `graphql-ws` protocol is not yet supported by GraphQL Playground or Apollo Explorer.